src/votekit/ballot_generator.py

```diff
@@ -8,6 +8,7 @@
 import pickle
 import random
 from typing import Optional
+import apportionment.methods as apportion
```

Comment:
Adding `# type: ignore` to end of this line should solve the mypy test failures.